todo-app/src/Components/todoList.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import { observer, inject } from "mobx-react";
+
+import TodoItem from "./todoItem";
+import "./styles/todoList.css";
+
+@inject("todoStore")
+@observer
+export default class TodoList extends Component {
+  componentDidMount() {
+    this.props.todoStore.loadTodos();
+  }
+
+  renderAllTodos = () => {
+    const todoItems = this.props.todoStore.todos;
+    return todoItems.map(item => {
+      return (
+        <TodoItem
+          key={item._id}
+          data={item}
+          handleToggleDone={this.handleToggleDone}
+        />
+      );
+    });
+  };
+
+  render() {
+    if (this.props.todoStore.loading) {
+      // return <img src="./loading.png" alt="loading" className="loadingImg" />; ////////////////////this is Not working WHY?
```

Comment:
Okay, fair question. The basic reason this doesn't work is that your `loading.png` image is inside your `src` dir, which is never exposed publicly. We aren't working with an old-school `index.html` with some css and images next to it in an exposed directory.

For CRA, and all modern projects, JavaScript, CSS, images, and pretty much any other asset you can think of are going to be handled, preprocessed, or transpiled via a build tool (typically Webpack at this moment in time).

That being said, your `loading.png` image _definitely_ should be inside your `src` dir as it's part of your "source code", despite being an image. Here's how:

    import loaderImage from './loading.png'
    <img src={loaderImage} alt="loading" className="loadingImg" />

Now, do this and then inspect the image on the browser page. Look at the `src` attribute. You'll see that there's a lot going on behind the scenes here.